src/assets/i18n/ja-JP.json

```diff
@@ -250,33 +250,33 @@
       "labelDim": "薄暗いラベル",
       "labelEnable": "ラベルを有効にする",
       "labelOffset": "ラベルの位置を上下に調整する",
-      "labelSize": "時計を基準にしてラベルのサイズを調整します",
+      "labelSize": "時計に対してラベルのサイズを調整します",
       "meridiem": "メリディアム",
       "meridiemDim": "薄暗いメリディアム",
       "meridiemEnable": "メリディアムを有効にする",
-      "meridiemOffset": "子午線の位置を上下に調整する",
-      "meridiemSize": "時計を基準にして子午線のサイズを調整します",
+      "meridiemOffset": "メリディアムの位置を上下に調整する",
+      "meridiemSize": "時計に対してメリディアムのサイズを調整します",
```

Comment:
This translated "meridiem" as "medium"

Reply:
It is hard to translate "meridiem" for me because we don't use it normally,
We just say the mark of a.m. or p.m. , 
and normally we say that it is 12-hour format or not.

If it is ok to translate it to "午前午後記号" as "the mark of a.m. and p.m."?

Reply:
Yes that would work fine for me, thanks